pkg/client/cache/imagestream.go

```diff
@@ -14,6 +17,19 @@ type StoreToImageStreamLister struct {
 	cache.Indexer
 }
 
+// Get returns the image stream with the provided namespace and name.
+func (s *StoreToImageStreamLister) Get(namespace, name string) (*imageapi.ImageStream, error) {
```

Comment:
Refactor to match client interface.  `.ImageStreams(namespace).Get(name)`.  I'm not ready to break that pattern yet.